stories/TextField.stories.svelte

```diff
@@ -9,12 +9,20 @@ const args = {
   label: title,
   class: '', //will only work with global class
 }
+const keydown = (event) => console.log('TextField keydown', event)
+const keypress = (event) => (lastKey = event.key)
+const keyup = (event) => console.log('TextField keyup', event)
```

Comment:
could you add these new methods to args like ` 'on:keydown': () => alert('you clicked'),` That is how they will get documented in storybook. Also I don't think you have a release triggering commit yet so make sure you commit message is `feat(TextField): add keypress, keydown and keyup`

Reply:
doh! i forgot the magic naming!!